src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -590,4 +593,71 @@ void notifySnapshotReady() {
         .snapshotReady(
             eq(id), anyString(), eq(snapshot.getName()), eq("No snapshot specification found"));
   }
+
+  @Test
+  void testGetRequestGroupMembers() {
+    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
+    expected.members(new ArrayList<>());
+    UUID requestId = new SnapshotAccessRequestResponse().getId();
+    SnapshotAccessRequestModel requestModel =
+        SnapshotBuilderTestData.createAccessRequestModelApproved();
+    when(snapshotRequestDao.getById(requestId)).thenReturn(requestModel);
+    when(iamService.getGroupPolicyEmails(requestModel.samGroupName(), IamRole.MEMBER.toString()))
+        .thenReturn(new ArrayList<>());
+    assertThat(snapshotBuilderService.getGroupMembers(requestId), is(expected));
+  }
+
+  @Test
+  void testAddRequestGroupMember() {
+    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
+    String memberEmail = "user@gmail.com";
+    expected.members(new ArrayList<>(List.of(memberEmail)));
```

Comment:
Do you need to create a new `ArrayList` here?
```suggestion
    String memberEmail = "user@gmail.com";
    expected.setMembers(List.of(memberEmail));
```